app/experimenter/nimbus-ui/src/components/FormBranches/FormBranch.tsx

```diff
@@ -0,0 +1,228 @@
+                variant="light"
+                className="bg-transparent border-0 p-0 m-0"
+                title="Remove branch"
+                onClick={handleRemoveClick}
+              >
+                <DeleteIcon width="18" height="18" />
+              </Button>
+            )}
+          </Form.Group>
+        </Form.Row>
+      </Form.Group>
+
+      {experimentFeatureConfig === null ? (
+        <Form.Group className="px-2 mx-3">
+          <Form.Row>
+            <Button
+              variant="outline-primary"
+              size="sm"
+              data-testid="feature-config-add"
+              onClick={onAddFeatureConfig}
+            >
+              + Feature configuration
+            </Button>
+          </Form.Row>
+        </Form.Group>
+      ) : (
+        <Form.Group
+          className="px-3 py-2 border-top"
+          data-testid="feature-config-edit"
+        >
+          <Form.Row>
+            <Col className="px-2">Feature configuration</Col>
+            <Form.Group as={Col} sm={1} className="align-top text-right">
+              <Button
+                variant="light"
+                className="bg-transparent border-0 p-0 m-0"
+                data-testid="feature-config-remove"
+                title="Remove feature configuration"
+                onClick={onRemoveFeatureConfig}
+              >
+                <DeleteIcon width="18" height="18" />
+              </Button>
+            </Form.Group>
+          </Form.Row>
+          <Form.Row className="align-middle">
+            <Form.Group as={Col} controlId={`${id}-feature`}>
+              <Form.Control
+                as="select"
+                data-testid="feature-config-select"
+                onChange={handleFeatureConfigChange}
+                value={featureConfig!.findIndex(
+                  (feature) =>
+                    experimentFeatureConfig !== null &&
+                    feature !== null &&
+                    feature.slug === experimentFeatureConfig.slug,
+                )}
+              >
+                <option value="">Select...</option>
+                {featureConfig?.map(
+                  (feature, idx) =>
+                    feature !== null && (
+                      <option
+                        key={`feature-${feature.slug}-${idx}`}
+                        value={idx}
+                      >
+                        {feature.name}
+                      </option>
+                    ),
+                )}
+              </Form.Control>
+            </Form.Group>
+            <Col sm={1} md={1} className="px-2 text-center">
+              is
+            </Col>
+            <Form.Group as={Col} controlId={`${id}-featureEnabled`}>
+              <Form.Check
+                type="switch"
+                label={featureEnabled ? "On" : "Off"}
+                defaultChecked={featureEnabled}
+                onChange={handleChange("featureEnabled", "checked")}
+              />
```

Comment:
Is this toggle also supposed to stay in sync with the other toggles? They seem to operate independently for me.

Reply:
No, this toggle is per-branch, since whether the feature is enabled or not can be part of the experiment branch